SpatialGDK/Source/SpatialGDKServices/Private/Interop/Connection/EditorWorkerController.cpp

```diff
@@ -1,16 +1,11 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
-#pragma once
+#include "EditorWorkerController.h"
 
-#if WITH_EDITOR
-
-#include "Editor.h"
-#include "Modules/ModuleManager.h"
-#include "Settings/LevelEditorPlaySettings.h"
-#include "SpatialGDKEditorToolbar.h"
+#include "Private/SpatialGDKServicesPrivate.h"
 
-namespace SpatialGDK
+#if WITH_EDITOR
+namespace
```

Comment:
How come this namespace is empty?

Reply:
To hide the implementation from the outer world.